Sometime error when running `test_mdscoring.py`

Sometimes, the uni-tests are failing, sometimes not.
related to `integration_tests/test_mdscoring.py`

Either a CNS rounding error or a Python rounding error

```
=========================== short test summary info ============================
FAILED integration_tests/test_mdscoring.py::test_mdscoring_default - assert [-42.645] == [-42.646]
  
  At index 0 diff: -42.645 != -42.646
  
  Full diff:
    [
  -     -42.646,
  ?           ^
  +     -42.645,
  ?           ^
    ]
```

--- a/integration_tests/test_mdscoring.py
+++ b/integration_tests/test_mdscoring.py
@@ -93,4 +93,5 @@ def test_mdscoring_default(mdscoring_module, calc_fnat):
     assert expected_interface_csv.exists(), f"{expected_interface_csv} does not exist"
     df_perint = pd.read_csv(expected_interface_csv, sep="\t", comment="#")
     # check that the score is equal to the global score (it's a dimer!)
-    assert df_perint["score"].tolist() == df["score"].tolist()
+    for perint, sc in zip(df_perint["score"].tolist(), df["score"].tolist()):
+        assert perint == pytest.approx(sc, abs=0.01)

